staking/types/validator.go

```diff
@@ -495,7 +495,11 @@ func UpdateValidatorFromEditMsg(validator *Validator, edit *EditValidator) error
 	}
 
 	if edit.Active != nil {
-		validator.Active = *edit.Active
+		if *edit.Active {
+			validator.Active = true
+		}
+	} else {
```

Comment:
this is wrong right? if I don't want to set the Active field, it will reset to false

Reply:
The RLP encoded EditValidator.Active only will have two value 0x1 or 0x80:

1. default is 0x80,   
2. 0x80 or if *Active == false
3. 0x1 if  *Active == true , you want to set true.

the default value is same as the false value...

Unless we change to *int,  otherwise *bool can't have 3 stage value ...